src/Mod/Sketcher/App/ConstraintPyImp.cpp

```diff
@@ -65,85 +65,105 @@ int ConstraintPy::PyInit(PyObject* args, PyObject* /*kwd*/)
     PyObject* oNumArg5;
     int any_index;
 
-    // ConstraintType, GeoIndex
-    if (PyArg_ParseTuple(args, "si", &ConstraintType, &FirstIndex)) {
+    PyObject* activated;
+    PyObject* driving;
+
+    Sketcher::Constraint* cstr = this->getConstraintPtr();

```

Comment:
I don't love this abbreviation (since it matches the very-commonly-used std::string::cstr). I'd spell out "constraint" here.

Reply:
Done